lib/cli.js

```diff
@@ -41,6 +41,16 @@ module.exports = function broccoliCLI(args) {
         options.environment = 'development';
       }
 
+      if (process.env.BROCCOLI_ENV && options.environment) {
+        throw new CliError('You can not use BROCCOLI_ENV and --environment/--prod/--dev together');
```

Comment:
I don't particularly like the idea of throwing here...

One should win (probably the `--environment` flag) over the other.

Reply:
I'm ok with that, `--environment` is the new one, so it can take precedence